Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -47,6 +47,12 @@ public override void Enable()
                 return;
             }
 
+            if (Controller != null)
```

Comment:
>Controller != null [](start = 16, length = 18)

What causes Enable() being called with a non-null controller? It seems that if Disable() is being called correctly this shouldn't happen.

Reply:
Disable isn't called necessarily between enable calls. Right now when our services and mrtk get set up we end up in a situation where we're enabling our services twice. I'm not sure what the intend behind this is. 

First time calling enable is when we're registering all our services (we're calling initialize and enable in that step) and after that the MRTK main class calls another EnableAllServices in it's OnEnable() method.